aldryn_newsblog/utils/utilities.py

```diff
@@ -174,6 +180,16 @@ def is_valid_namespace_for_language(namespace, language_code):
         return is_valid_namespace(namespace)
 
 
+def get_valid_languages_from_request(namespace, request):
+    language = translation.get_language_from_request(
+        request, check_path=True)
+    site_id = getattr(get_current_site(request), 'id', None)
```

Comment:
I'm a bit curious as I've seen this in some other addons.. what causes the site to be none?
Django won't actually work if there's no configured site, it will throw an ImproperlyConfiguredError